appender-core/src/main/java/com/van/logging/aws/S3PublishHelper.java

```diff
@@ -46,10 +47,14 @@ public S3PublishHelper(S3Configuration s3, boolean verbose) {
 
         this.bucket = s3.getBucket().toLowerCase();
         String path = s3.getPath();
-        if (!path.endsWith("/")) {
-            this.path = path + "/";
+        if (path != null && !path.isEmpty()) {
```

Comment:
Also try using `StringUtils.isTruthy(path)` here.

Reply:
Incorporated the change